syncd/Syncd.cpp

```diff
@@ -4395,7 +4396,8 @@ void Syncd::performWarmRestartSingleSwitch(
         SAI_SWITCH_ATTR_NAT_EVENT_NOTIFY,
         SAI_SWITCH_ATTR_PORT_STATE_CHANGE_NOTIFY,
         SAI_SWITCH_ATTR_QUEUE_PFC_DEADLOCK_NOTIFY,
-        SAI_SWITCH_ATTR_BFD_SESSION_STATE_CHANGE_NOTIFY
+        SAI_SWITCH_ATTR_BFD_SESSION_STATE_CHANGE_NOTIFY,
+        SAI_SWITCH_ATTR_TWAMP_SESSION_EVENT_NOTIFY
```

Comment:
this entire list could be obtained from sai_metadata_switch_notify_attr, and there will be no need in the future to update that, i will make PR for this after your will merge

Reply:
Got it.  Whether I would do nothing before you make PR.

Reply:
go on proceeded with yours, i will add my on top of it